src/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,104 @@
+@import "../../assets/variables.scss";
+
+.navbar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: $nav-height;
+  background-color: $primary001;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: $M-spacing;
+}
+
+.hamburgerButton {
+  display: flex;
+  align-items: center;
+  justify-items: center;
+  cursor: pointer;
+  width: 3.2rem;
+  height: 3rem;
+  background-color: transparent;
+  border: none;
+}
+
+.hamburger {
+  display: inline-block;
+  height: 0.2rem;
+  width: 100%;
+  background-color: $white;
+  transform: translateY(-50%);
+  transition: background-color 0.2s 0.2s ease-in-out;
+  &::before,
+  &::after {
+    content: "";
+    position: absolute;
+    display: block;
+    height: 0.2em;
+    width: 100%;
+    background-color: $white;
+    transition: transform 0.2s 0.2s ease-in-out;
+  }
+  &::before {
+    top: 10px;
+  }
+  &::after {
+    top: -10px;
+  }
+  &-active {
```

Comment:
One trick I used to use with CSS modules it similar cases was adding another class like
```suggestion
  &.active {
```
and thanks to the fact that the classes are concatenated with `&` I could use
```jsx
        <span
          className={cn(styles.hamburger, {
            [styles.active]: isOpen,
          })}
        ></span>
```
in JSX which could simplify the naming a little bit. Similar solution can be used with `.menuList` and thanks to that the classes are "concatenated", there will be no conflicts and it will work for both cases - `.hamburger` and `.menuList`

Reply:
Great idea, I totally forgot about this way of writing css selector 😁